app.js

```diff
@@ -220,3 +220,18 @@ for (let i = 0; i < projbtnD.length; i += 1) {
     fillData2(popUpDatasD[i]);
   });
 }
+
+// form validation
+
+const emails = document.forms[0].elements[1];
+const subBtn = document.querySelector('button.bom');
+const errMsg = document.querySelector('form > .errmsg');
+
+subBtn.addEventListener('click', (event) => {
+  const { value } = emails;
+  if (value !== value.toLowerCase()) {
+    event.preventDefault();
+
+    errMsg.textContent = 'kindly enter lowerCase character in the email field!';
+  }
+});
```

Comment:
- [ ] Kindly make sure you validate your form to check that the email is written in all lower case. Right now it doesn't work.

[Video](https://www.awesomescreenshot.com/video/10111970?key=f229dca057438d5c0b2a9d5b35126182)

Reply:
I forget the desktop validation!

Reply:
Done